Quasardb.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29521.150
+# Visual Studio Version 17
```

Comment:
Does this have any implications for our customers? I know nothing about .NET, does this mean they need VS17 to compile our code?

Reply:
I am expert with the same level as you )
And can return this version back.
I don't know is it impact to users
I changed this because my local VC 16 doesn't support this project.

Reply:
Maybe someone else knows? Maybe we should exclude this from the commit if we don't understand the implications.